### DISCUSSION TITLE
Patch release during semester break?

### DISCUSSION THREAD

**Discussion by ralf-berger** [ude-soco/CourseMapper-webserver#993] [General]
There are a bunch of dependency updates in `main`, some of them major. Is there a suitable time for a patch release during the semester break?

**Comment by jeanqussa** [comment]
We would like to merge human-in-the-loop and development into main before doing a release. Or would that be more than a patch release?

**Reply by ralf-berger** [reply]
That sounds like more like what's assigned to the next [major release (4.0.0)](https://github.com/ude-soco/CourseMapper-webserver/milestone/3).
